translations/heroes.tex/pl.po

```diff
@@ -81,6 +84,14 @@ msgid ""
 "Gaining 1 Experience\\index{Experience} is represented by the symbol \\includesvg[height=10px]{\\svgs/experience.svg}.\n"
 "\n"
 msgstr ""
+"\\subsection*{Bohater Główny}\n"
+"The Bohater główny\\index{Bohater główny} reprezentowany jest przez figurkę, Planszetkę Bohatera i twoją Talię.\n"
+"W każdej Frakcji Bohater Główny ma do wydania 3 Punkty Ruchu.\n"
```

Comment:
"W każdej Frakcji Bohater Główny ma do wydania 3 Punkty Ruchu (3 PR).\n"

Reply:
(PR) już zostaje wprowadzone w Przebiegu Rundy - czy potrzebujemy też tego zwinięcia/rozwinięcia tutaj?

Reply:
wydaje mi się, że tą fanowską ksiege zasad mozna czytac fragmentami tzn (poszczególnymi sekcjami) i wprowadzenie skrótowca powinno być w każdej cześci, ale to tylko moje ksiązkowe podejscie...

Reply:
makes sense